examples/evaluation_and_profiling/swe_bench/tests/test_iterative_predictor.py

```
@@ -0,0 +1,700 @@
+        agent = IterativeAgent(mock_llm, repo_path, config)
+
+        with patch('nat_swe_bench.predictors.predict_iterative.predict_iterative.asyncio.to_thread') as mock_thread:
+            # Return realistic outputs for each command
+            mock_thread.side_effect = [
+                MagicMock(returncode=0, stdout="main.py\nREADME.md\n"),
+                MagicMock(returncode=0, stdout="old code here\n"),
+                MagicMock(returncode=0, stdout=""),
+                MagicMock(returncode=0, stdout="COMPLETE_TASK_AND_SUBMIT_FINAL_OUTPUT\ndiff --git a/main.py\n-old\n+new"),
+            ]
+
+            exit_status, patch_result = await agent.run("Fix the bug in main.py")
```

Ruff (0.14.14)

Unpacked variable `patch_result` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)